Tools/yocto/meta-openembedded_and_meta-webkit.patch

```diff
@@ -25,20 +25,6 @@ index f4911f3..b13abad 100644
  
  [Install]
  WantedBy=multi-user.target
-diff --git a/sources/meta-webkit/recipes-browser/packagegroups/packagegroup-wpewebkit-depends.bb b/sources/meta-webkit/recipes-browser/packagegroups/packagegroup-wpewebkit-depends.bb
-index 9459f4c..e74a20c 100644
---- a/sources/meta-webkit/recipes-browser/packagegroups/packagegroup-wpewebkit-depends.bb
-+++ b/sources/meta-webkit/recipes-browser/packagegroups/packagegroup-wpewebkit-depends.bb
-@@ -170,9 +170,6 @@ RDEPENDS:packagegroup-wpewebkit-depends-desktop = "\
-     libxtst \
-     libx11-locale \
-     xorg-minimal-fonts \
--    gdk-pixbuf-loader-ico \
```

Comment:
This was necessary on my system to build for rpi3 - last time I tried this, four weeks ago. I think we should keep it for now.

Reply:
Self-reply: just saw that you updated meta-webkit - I assume my patch was already merged upstream.

Reply:
yes, it was merged